codes/go/chapter_computational_complexity/leetcode_two_sum_test.go

```diff
@@ -0,0 +1,19 @@
+package chapter_computational_complexity
+
+import (
+	"testing"
+)
+
+func TestTwoSum(t *testing.T) {
```

Comment:
Sorry, I'm not familiar with Go.
Can we merge the two files, `leetcode_two_sum_test.go` and `leetcode_two_sum.go` ?

Reply:
Go has a built-in testing command called `go test` and a package `testing` which combine to give a minimal but complete testing experience.So the two files are usually separated.

Reply:
Thanks for the clear explanation.